src/style-util.js

```diff
@@ -291,4 +291,39 @@ export function gatherStyleText(element) {
     }
   }
   return styleTextParts.join('').trim();
+}
+
+/**
+ * Split a selector separated by commas into an array in a smart way
+ * @param {string} selector
+ * @return {!Array<string>}
+ */
+export function splitSelectorList(selector) {
+  let parts = [];
+  let part = '';
+  for (let i = 0; i >= 0 && i < selector.length; i++) {
```

Comment:
Would `i` ever be less than 0?

Reply:
(from offline) `findMatchingParen` might return -1 and `i = end;` happens unconditionally after it.